### ISSUE TITLE
Could not initialize random number generator

### ISSUE THREAD

**Issue by maartenhendrix** [monicahq/docker#107]
When i try to install monicahq in docker on my synology i get the next error:
`Database ready.[Mon May 09 12:28:40.199417 2022] [:crit] [pid 1] (38)Function not implemented: AH00141: Could not initialize random number generator`
Could someone please point me in a direction in how to solve this?

``` 
✓ Maintenance mode: on
'/usr/local/bin/php' 'artisan' down --retry="10"
Application is now in maintenance mode.
✓ Resetting application cache
'/usr/local/bin/php' 'artisan' cache:clear
Application cache cleared!
✓ Clear config cache
'/usr/local/bin/php' 'artisan' config:clear
Application key set successfully.
Configuration cache cleared!
✓ Clear route cache
'/usr/local/bin/php' 'artisan' route:clear
Route cache cleared!
✓ Clear view cache
'/usr/local/bin/php' 'artisan' view:clear
Compiled views cleared!
✓ Performing migrations
'/usr/local/bin/php' 'artisan' migrate --force
Nothing to migrate.
✓ Check for encryption keys
'/usr/local/bin/php' 'artisan' monica:passport --force
Checking encryption keys...
✓ Files storage/oauth-private.key and storage/oauth-public.key detected.
Checking Personal Access Client...
✓ Personal Access Client already created.
✓ Ping for new version
'/usr/local/bin/php' 'artisan' monica:ping --force
Call url: https://version.monicahq.com/ping
instance version: 3.7.0
current version: 3.7.0
✓ Maintenance mode: off
'/usr/local/bin/php' 'artisan' up
Application is now live.
Monica v3.7.0 is set up, enjoy.
Database ready.[Mon May 09 12:28:40.199417 2022] [:crit] [pid 1] (38)Function not implemented: AH00141: Could not initialize random number generator
```

**Comment by maartenhendrix** [comment]
I will try if this will fix it https://github.com/nextcloud/docker/issues/1574#issuecomment-923090766

**Comment by maartenhendrix** [comment]
> I will try if this will fix it [nextcloud/docker#1574 (comment)](https://github.com/nextcloud/docker/issues/1574#issuecomment-923090766)

this worked for me.

**Comment by slifty** [comment]
I suppose I'm confused... but why would the kernel of DSM impact the functionality of the docker container?
